examples/grpc/grpc-streaming-client/main.go

```diff
@@ -0,0 +1,242 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"time"
+
+	"gofr.dev/examples/grpc/grpc-streaming-client/client"
+	"gofr.dev/pkg/gofr"
+)
+
+func main() {
+	app := gofr.New()
+
+	// Create a gRPC client for the Chat Streaming service
+	chatClient, err := client.NewChatServiceGoFrClient(app.Config.Get("GRPC_SERVER_HOST"), app.Metrics())
+	if err != nil {
+		app.Logger().Errorf("Failed to create Chat client: %v", err)
+	}
+
+	chat := NewChatHandler(chatClient)
+
+	app.GET("/chat/server-stream", chat.ServerStreamHandler)
+	app.POST("/chat/client-stream", chat.ClientStreamHandler)
+	app.GET("/chat/bidi-stream", chat.BiDiStreamHandler)
+
+	app.Run()
+}
+
+type ChatHandler struct {
+	chatClient client.ChatServiceGoFrClient
+}
+
+func NewChatHandler(chatClient client.ChatServiceGoFrClient) *ChatHandler {
+	return &ChatHandler{chatClient: chatClient}
+}
+
+type StreamResponse struct {
+	Message   string    `json:"message"`
+	Timestamp time.Time `json:"timestamp"`
+	Direction string    `json:"direction"` // "received" or "sent"
+}
+
+// ServerStreamHandler handles server-side streaming with detailed response tracking
+func (c *ChatHandler) ServerStreamHandler(ctx *gofr.Context) (interface{}, error) {
+	startTime := time.Now()
+	var responses []StreamResponse
+
+	// Record initial request
+	responses = append(responses, StreamResponse{
+		Message:   "initiating server stream request",
+		Timestamp: startTime,
+		Direction: "sent",
+	})
+
+	stream, err := c.chatClient.ServerStream(ctx, &client.Request{Message: "stream request"})
+	if err != nil {
+		return nil, fmt.Errorf("failed to initiate server stream: %v", err)
+	}
+
+	// Handle server streaming
+	for {
+		res, err := stream.Recv()
+		if err != nil {
+			if errors.Is(err, io.EOF) {
+				break
+			}
+			return nil, fmt.Errorf("stream receive error: %v", err)
+		}
+
+		// Record received message
+		response := StreamResponse{
+			Message:   res.Message,
+			Timestamp: time.Now(),
+			Direction: "received",
+		}
+		responses = append(responses, response)
+		ctx.Logger.Infof("Received server stream message: %s at %v", res.Message, response.Timestamp)
+	}
+
+	// Return detailed stream information
+	return map[string]interface{}{
```

Comment:
```suggestion
	return map[string]any{
```

Here and everywhere in fact